docs/source/high-level-arch.rst

```diff
@@ -0,0 +1,56 @@
+.. meta::
+   :description: TorchRec High Level Architecture
+   :keywords: recommendation systems, sharding, distributed training, torchrec, architecture
+
+==================
+TorchRec High Level Architecture
+==================
+
+In this section, you will learn about the high-level architecture of TorchRec, designed to optimize large-scale recommendation systems using PyTorch. You will learn how TorchRec employs model parallelism to distribute complex models across multiple GPUs, enhancing memory management and GPU utilization, as well as get introduced to TorchRec's base components and sharding strategies.
```

Comment:
You probably want to have the lines to wrap at ~79 symbols or so.

Reply:
thanks, formatted using rstfmt